modules/ROOT/pages/create-tokenization-service.adoc

```diff
@@ -0,0 +1,37 @@
+= Create a Tokenization Service
+
+The tokenization service enables you to substitute a sensitive data element with a non-sensitive equivalent.
+
+== Prerequisites
+
+To configure and use the tokenization service, you need the following:
+
+* Runtime Fabric 1.1.153 or later with inbound traffic configured. See xref:1.2@runtime-fabric::index.adoc[the Runtime Fabric documentation].
```

Comment:
If you aren't going to give a link to the topic where you configure inbound traffic, I'd move the link to See Also section. You don't really want people bailing out of the topic at this point. I'd move all the links in this list to See Also.